ui/ducks/metamask/metamask.test.js

```diff
@@ -480,32 +307,6 @@ describe('MetaMask Reducers', () => {
     expect(state.pendingTokens).toStrictEqual({});
   });
 
-  it('update ensResolution', () => {
-    const state = reduceMetamask(
-      {},
-      {
-        type: actionConstants.UPDATE_SEND_ENS_RESOLUTION,
-        payload: '0x1337',
-      },
-    );
-
-    expect(state.send.ensResolution).toStrictEqual('0x1337');
-    expect(state.send.ensResolutionError).toStrictEqual('');
-  });
-
-  it('update ensResolutionError', () => {
```

Comment:
These tests were removed instead of relocated because they will eventually be obsolete in #10965